src/fragments/lib/auth/flutter/getting_started/40_authenticator.mdx

```diff
@@ -0,0 +1,271 @@
+## Authentication with Amplify
+
+There are 2 options to add authentication capabilities to your application:
+- Option 1: Use pre-built UI components
```

Comment:
I think that a) we should use the term "Authenticator" when describing option 1, and b) we should probably call it out as in Developer Preview?

Reply:
I have made a change to the title - and also added a callout for developer preview @haverchuck